cmd/tailscale/cli/set.go

```diff
@@ -76,6 +79,11 @@ func newSetFlagSet(goos string, setArgs *setArgsT) *flag.FlagSet {
 	setf.BoolVar(&setArgs.updateApply, "auto-update", false, "automatically update to the latest available version")
 	setf.BoolVar(&setArgs.postureChecking, "posture-checking", false, "HIDDEN: allow management plane to gather device posture information")
 	setf.BoolVar(&setArgs.runWebClient, "webclient", false, "run a web interface for managing this node, served over Tailscale at port 5252")
+	dap := tailfs.DefaultAutomountPath()
+	if !version.IsSandboxedMacOS() && dap != "" {
+		setf.BoolVar(&setArgs.automountEnabled, "automount-enabled", false, "enable automatically mounting Tailscale shares")
+		setf.StringVar(&setArgs.automountPath, "automount-path", "", fmt.Sprintf(`path at which to automount shares, leave blank to default to %q`, dap))
```

Comment:
is it just one Tailscale mount? (one big VFS with the top-level directories between peers, then their shares, etc?)

Reply:
Yes, it's one big VFS, with the following structure:

`<mount point>/<tailnet>/<peer>/<share>`